pkg/plugins/policies/core/matchers/dataplane_test.go

```diff
@@ -16,8 +16,9 @@ import (
 	"github.com/kumahq/kuma/pkg/core/resources/registry"
 	_ "github.com/kumahq/kuma/pkg/plugins/policies"
 	"github.com/kumahq/kuma/pkg/plugins/policies/core/matchers"
+	mal_api "github.com/kumahq/kuma/pkg/plugins/policies/meshaccesslog/api/v1alpha1"
 	"github.com/kumahq/kuma/pkg/plugins/policies/meshhttproute/api/v1alpha1"
-	policies_api "github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission/api/v1alpha1"
+	mtp_api "github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission/api/v1alpha1"
```

Comment:
I remember in other places we are using full policy names instead of abbreviations in imports. I know aliases are very long then, but with these abbreviations, whenever I'm reading a code containing them, it takes me few additional seconds to decipher them.

Should we be consistent and instead of:
- `mal_api` do `meshaccesslog_api`;
- `mtp_api` do `meshtrafficpermission_api`;